.travis.yml

```diff
@@ -7,4 +7,5 @@ rvm:
   - 2.2.0
   - 2.2.1
   - 2.2.2
+  - 2.3.0
```

Comment:
It's the [ruby-version](https://github.com/rapid7/convection/blob/master/.ruby-version) we specify.